app/views/content/terms_candidate.html.erb

```diff
@@ -7,75 +7,90 @@
     <p class="govuk-body">These terms and conditions set out the standards the Department for Education (DfE) expects when you use this service to: </p>
 
     <ul class="govuk-list govuk-list--bullet">
-      <li>submit an application to initial teacher training courses</li>
-      <li>make contact with other people using this service</li>
+      <li>apply to initial teacher training courses</li>
+      <li>contact other people using this service</li>
       <li>link to this service, or interact with it in any other way</li>
     </ul>
 
     <p class="govuk-body">Applying for teacher training using this service means you agree to these terms. If you do not agree, you must stop using the service.</p>
 
-    <p class="govuk-body">If you do not follow them, your application might be withdrawn and your account might be blocked.</p>
+    <p class="govuk-body">If you do not follow these terms and conditions, your application might be withdrawn and your account might be blocked.</p>
 
     <p class="govuk-body">Using this service also means you agree to the <%= govuk_link_to('GOV.UK terms and conditions', t('govuk.terms_conditions_url')) %>.</p>
 
     <p class="govuk-body">Everything you include in your application form must be truthful and accurate.</p>
 
-    <p class="govuk-body">It’s your responsibility to keep your account details safe. You should not share them with anyone else.</p>
+    <p class="govuk-body">When you use this service, you will create an account. It is your responsibility to keep your account details safe. You should not share them with anyone else.</p>
 
     <h2 class="govuk-heading-m">Prohibited uses</h2>
 
     <p class="govuk-body">You must not:</p>
 
     <ul class="govuk-list govuk-list--bullet">
-      <li>use our service for illegal or fraudulent reasons</li>
-      <li>gather or use data from our service</li>
-      <li>use abusive language or act abusively towards anyone you come into contact with through using this service</li>
+      <li>use the service for illegal or fraudulent reasons</li>
+      <li>gather or use data from the service</li>
+      <li>use abusive language or act abusively towards anyone you contact using this service</li>
```

Comment:
I worry this potentially weakens our standing if someone is abusive to someone in, say, an interview with a provider. If the provider fed this back, what would our recourse be to block this person? 

The former version gives us remit to take action, as they came into contact through the use of the service (applying and then attending the interview), whereas this new version risks not being able to, as the candidate doesn't use the service to directly contact the person.

What do you think?

Reply:
Sounds good, have updated